src/octopeer-github/test/resources/Commit_tab.html

```diff
@@ -0,0 +1,1008 @@
+
+
+
+
+<!DOCTYPE html>
```

Comment:
Can you consider de-capitalizing the filenames for the html files?
E.g. `Commit_tab.html` -> `commit-tab.html`